assets/css/components/button.css

```diff
@@ -1,27 +1,25 @@
 .mbta-button {
   align-items: center;
-  border-radius: var(--border-radius-md);
-  border-width: var(--border-width-sm);
+  border-radius: var(--border-radius-default);
+  border-width: var(--border-width-default);
   column-gap: var(--spacing-sm);
   display: inline-flex;
   font-family: var(--font-family-base);
   font-size: var(--font-size-default);
   font-weight: 500;
+  line-height: var(--line-height-default);
   padding-bottom: var(--spacing-sm);
   padding-top: var(--spacing-sm);
-  padding-left: var(--spacing-md);
-  padding-right: var(--spacing-md);
-  transition: all var(--transition-duration-default) ease;
+  padding-left: var(--spacing-default);
+  padding-right: var(--spacing-default);
+  transition: background-color var(--transition-duration-default) ease;
```

Comment:
This isn't introduced in this PR, but it seems relevant to the current work: `--transition-duration-default` isn't defined, which means that the transition isn't animating - it's just snapping to the hover color. The variable `--transition-duration` _is_ defined, and a 0.2s color transition actually looks quite nice 🙂